orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -81,21 +87,47 @@ run_azure_backup() {
   rmdir -v "$DATE_DIR" > >(tee -a $LOGFILE) 2> >(tee -a $LOGFILE >&2)
 }
 
+replace_ranger_group_before_export() {
+  doLog "INFO Replacing "$1" with RANGER_WAG in the dump before export"
+  echo "sed --in-place="original" 's/"$1"/RANGER_WAG/g' $2"
+  ret_code=$(sed --in-place="original" -e s/"$1"/RANGER_WAG/g "$2" || echo $?)
```

Comment:
**question:** How confident are we that `RANGER_WAG` will never be in the SQL files that we're using?

Is there a string we could use here that is both _invalid SQL_ (preventing restoration unless replacement occurs) and that we can be sure won't show up as part of the dump?

Reply:
Will "RANGER_WAG" with a string that will be unique.

Implementing logic to prevent restoration unless replacement occurs needs further though. Let's not hold the patch for it.

Reply:
agreed!